admin/class-extensions.php

```diff
@@ -20,26 +20,31 @@ class WPSEO_Extensions {
 			'slug'       => 'yoast-seo-premium',
 			'identifier' => 'wordpress-seo-premium',
 			'classname'  => 'WPSEO_Premium',
+			'my-yoast'   => WPSEO_Addon_Manager::PREMIUM_SLUG,
```

Comment:
Please rename 'my-yoast' to 'my-yoast-slug' or something that is more descriptive. It's not a MyYoast 😉